src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -143,12 +150,12 @@ export const OpportunityCard = ({
                 <span className="text-heading-2 text-black hyphens-auto">{name}</span>
               </div>
 
-              <div className="relative top-0 right-0">
+              <div>
                 <SelectedCheckbox selected={selected} toggleSelection={toggleSelection} name={name} />
               </div>
             </div>
           )}
-          <span className="font-arial text-body-md">{description}</span>
+          <span className="font-arial text-[15px] leading-[23px] sm:text-body-md">{description}</span>
```

Comment:
I've asked from design about how to handle the "non-standard" text sizes, and the reply was to use the closest that match the design token. In this case the text size should be 16px and leading 24px, so basically the `text-body-md`

Reply:
https://www.figma.com/design/6M2LrpSCcB0thlFDaQAI2J/cx_jod_client?node-id=2225-31563&node-type=frame&m=dev#949986837